README.md

```diff
@@ -14,16 +14,16 @@
 [![.NET Publish](https://github.com/Founntain/osuplayer/actions/workflows/dotnet-publish.yml/badge.svg)](https://github.com/Founntain/osuplayer/actions/workflows/dotnet-publish.yml)
  
 osu!player is a music player for *osu!* with the focus of playing your osu! songs **without having to start osu!**.
-This is the *official* **osu!player** repository and is mainly developed by [@Founntain](https://github.com/Founntain), with help of [@Cesan](https://github.com/Cesan).  
+This is the *official* **osu!player** repository and is mainly developed by [@Founntain](https://github.com/Founntain), with the help of [@Cesan](https://github.com/Cesan).  
 
-If you want to contribute, feel free to fork this repository, read the [contributing information](https://github.com/osu-player/osuplayer#-contributing-to-the-project) and head to the [official osu!player discord](https://discord.gg/RJQSc5B) (not mandetory, but helpful). There you can add all valuable ideas and discuss other stuff regarding development.
+If you want to contribute, feel free to fork this repository, read the [contributing information](https://github.com/osu-player/osuplayer#-contributing-to-the-project) and head to the [official osu!player discord](https://discord.gg/RJQSc5B) (not mandatory, but helpful). There you can add all valuable ideas and discuss other stuff regarding development.
 
 ## ☝️ Requirements
 
 #### osu!player requirements
 ✔️ A working computer  
 ✔️ .NET 6 or later installed  
-✔️ osu! installed with an **osu!.db file** or **osu!lazer client.realm** *(Beatmaps imported in osu!)*  
+✔️ osu! installed with an **osu!.db file** or **osu!lazer client.realm** *(Beat maps imported in osu!)*  
```

Comment:
"Beatmaps" is a term of osu! and I would rather write this as one word instead of two.

Reply:
I WAS FASTER HAHAHAHA